casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -826,12 +828,24 @@ object BlockAPI {
                            .pure[F]
                        case None =>
                          for {
-                           lastFinalizedBlock <- casper.lastFinalizedBlock
-                           runtimeManager     <- casper.getRuntimeManager
-                           postStateHash      = ProtoUtil.postStateHash(lastFinalizedBlock)
-                           res                <- runtimeManager.playExploratoryDeploy(term, postStateHash)
-                           lightBlockInfo     <- getLightBlockInfo[F](lastFinalizedBlock)
-                         } yield (res, lightBlockInfo).asRight[Error]
+                           targetBlock <- if (blockHash.isEmpty)
```

Comment:
@zsluedem How about making blockHash Option[String] with default None? Otherwise in the code to query state of LFB we have to do `exploratoryDeploy(term, "")` -> the purpose of "" is not really obvious. Better to have 
`exploratoryDeploy(term)` when querying LFB state and `exploratoryDeploy(term, hash)` for  some particular block